routing/router.go

```diff
@@ -1918,7 +1936,7 @@ func (r *ChannelRouter) processSendError(paySession PaymentSession,
 		processChannelUpdateAndRetry(
 			&onionErr.Update, errSource,
 		)
-		return false
+		return false, 0
```

Comment:
are all of these 0's meant to be `channeldb.FailureReasonTimeout`?

Reply:
No, the `0` values are irrelevant, because the first return parameter is `false`. I first had a single return parameter `*channeldb.FailureReason`, but it got ugly with obtaining pointers to consts.